frontend/packages/client/src/pages/Community.js

```diff
@@ -265,7 +265,16 @@ export default function Community() {
   return (
     <section className="full-height pt-0">
       {community ? (
-        <div className="is-flex community-header-wrapper">
+        <div
+          className="is-flex community-header-wrapper"
+          style={{
+            backgroundImage: community?.bannerImgUrl
+              ? `url(${community.bannerImgUrl})`
+              : 'none',
+            backgroundSize: 'cover',
+            backgroundPosition: 'center',
+          }}
+        >
           <div className={headerContainerClassNames}>
```

Comment:
I think this might be more a design question but depending on the background image tabs will kind of look faded away:
<img width="451" alt="Screen Shot 2022-07-21 at 10 52 50" src="https://user-images.githubusercontent.com/7526740/180236971-048e359c-6e14-4be6-8177-a061fbad5d93.png">